app/src/main/java/app/gamenative/service/SteamService.kt

```diff
@@ -2468,6 +2478,9 @@ class SteamService : Service(), IChallengeUrlChanged {
                     }
             }
         }
+
+        // Fetch and cache playtime data for PICS processing
+        fetchAndCachePlaytimeData()
```

Comment:
Grabbing this on start of the session. It's cheaper performance-wise to do this here. 

The other way we could do it is expose set the cache within the getOwnedGames() function. It would mean that we have 2 concerns in one function though.